### ISSUE TITLE
Versions of installed libraries and packages

### ISSUE THREAD

**Issue by TashinAhmed** [adamjking3/Stock-Trading-Environment#13]
Hi, is it possible to provide the versions of installed libraries and frameworks?
It seems the versions used in this codebase is depricated. 
Thanks.
@notadamking

**Comment by jeanloubet** [comment]
Hi, 
It work for me with (with python 3.7)

Package              Version
-------------------- ---------
absl-py              1.0.0
ale-py               0.7.4
astor                0.8.1
cached-property      1.5.2
certifi              2021.10.8
charset-normalizer   2.0.12
cloudpickle          2.0.0
cycler               0.11.0
fonttools            4.31.1
gast                 0.5.3
google-pasta         0.2.0
grpcio               1.44.0
gym                  0.11.0
gym-notices          0.0.6
h5py                 3.6.0
idna                 3.3
importlib-metadata   4.11.3
importlib-resources  5.4.0
joblib               1.1.0
Keras-Applications   1.0.8
Keras-Preprocessing  1.1.2
kiwisolver           1.4.0
Markdown             3.3.6
matplotlib           3.5.1
numpy                1.21.5
opencv-python        4.5.5.64
packaging            21.3
pandas               1.3.5
Pillow               9.0.1
pip                  21.2.2
protobuf             3.19.4
pygame               2.1.0
pyglet               1.5.23
pyparsing            3.0.7
python-dateutil      2.8.2
pytz                 2022.1
requests             2.27.1
scipy                1.7.3
setuptools           58.0.4
six                  1.16.0
stable-baselines     2.10.2
tensorboard          1.14.0
tensorflow           1.14.0
tensorflow-estimator 1.14.0
termcolor            1.1.0
typing_extensions    4.1.1
urllib3              1.26.9
Werkzeug             2.0.3
wheel                0.37.1
wrapt                1.14.0
zipp                 3.7.0
